drizzlepac/runastrodriz.py

```diff
@@ -550,6 +551,7 @@ def process(inFile, force=False, newpath=None, num_cores=None, inmemory=True,
         # Generate headerlets for each updated FLT image
         hlet_msg = _timestamp("Writing Headerlets started")
         for fname in _calfiles:
+            print("Creating new headerlet from {}".format(fname))
```

Comment:
Convert this to a log message or delete.

Reply:
Addressed in 8514ba1